flows/contact.go

```diff
@@ -143,14 +157,34 @@ func (c *Contact) Language() envs.Language { return c.language }
 // Blocked returns whether the contact is blocked or not
 func (c *Contact) Blocked() bool { return c.blocked }
 
-// SetBlocked sets the blocked state of this contact
-func (c *Contact) SetBlocked(blocked bool) { c.blocked = blocked }
-
 // Stopped returns whether the contact is stopped or not
 func (c *Contact) Stopped() bool { return c.stopped }
 
-// SetStopped sets the stopped stte of this contact
-func (c *Contact) SetStopped(stopped bool) { c.stopped = stopped }
+// Status returns the contact status
+func (c *Contact) Status() ContactStatus {
+	if c.stopped {
```

Comment:
replace `c.stopped` and `c.blocked` with `c.status`.. `Contact`  here doesn't need to reflect what we currently have in the RP database.. which is almost certainly going to change